src/apache/conf/ssl.conf

```diff
@@ -161,3 +161,17 @@ SSLRandomSeed connect file:/dev/urandom 512
         Header always set Strict-Transport-Security "max-age=63072000; includeSubdomains;"
     </IfDefine>
 </VirtualHost>
+
+#########################
+#        Snapweb        #
+#########################
+Listen 4201
+<VirtualHost *:4201>
```

Comment:
Obviously snapweb is already listening (with no SSL) on 4200. What is the purpose of also having it here? Just to add SSL to it?